src/Symlex/Router/TwigRouter.php

```diff
@@ -99,11 +99,12 @@ public function route($routePrefix = '', $servicePrefix = 'controller.web.', $se
             return $response;
         };
 
-        $indexRequestHandler = function (Request $request) use ($app, $container, $servicePrefix, $servicePostfix, $webRequestHandler) {
-            return $webRequestHandler('index', $request, 'index');
+        $indexRequestHandler = function (Request $request, $action='index') use ($app, $container, $servicePrefix, $servicePostfix, $webRequestHandler) {
+            return $webRequestHandler('index', $request, $action);
         };
 
         $app->get($routePrefix . '/', $indexRequestHandler);
+        $app->get($routePrefix . '/{action}', $indexRequestHandler)->assert('action', '.+');
```

Comment:
Wouldn't this prevent the rule in the next line from working? Looks like the same pattern or what would be the difference?

```
$routePrefix . '/{action}'
$routePrefix . '/{controller}'
```